pkg/kconfig/reduce.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package kconfig
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/google/syzkaller/pkg/debugtracer"
+)
+
+// Reduce finds an equivalent smaller config with respect to the `pred` predicate.
```

Comment:
I think we should improve/replace Minimize instead if we are not going to use it.
Also run the current Minimize tests again the new implementation (do they still pass?).